src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -45,24 +46,23 @@ public GitHubPackage(IServiceProvider serviceProvider)
             this.serviceProvider = serviceProvider;
         }
 
-        protected override void Initialize()
+        protected override async tasks.Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
-            base.Initialize();
-            IncrementLaunchCount();
+            var usageTracker = await GetServiceAsync(typeof(IUsageTracker)) as IUsageTracker;
+            await tasks.Task.Run(() => usageTracker.IncrementLaunchCount());
```

Comment:
Correct, I just left a similar comment both for async services and packages, the initialize methods should always be called in background threads.